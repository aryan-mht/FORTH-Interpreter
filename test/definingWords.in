."PI" 3.14 constant 
."area" : dup * PI ; 
."circum" : 2 * PI * ;
10 area . 
10 circum .  
