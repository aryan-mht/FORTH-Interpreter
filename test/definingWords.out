314.1526
62.8362
